### DISCUSSION TITLE
PayPal donation allowed?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#42296] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I know that GitHub Pages are for personal non-commercial pages.

Does GitHub's policies allow embedding a PayPal donation button/link on the web-page?

**Comment by samharp** [comment]
I will start off by saying this is not legal advice (as I'm not a lawyer), but I was unable to find wording within the GitHub Pages documentation or the GitHub Terms of Service that specifically limits the use of a PayPal donate button on GitHub Pages. Which, if I may say, makes sense as open source developers have been collecting payments via PayPal for years (well before GitHub Pages or Sponsors). They do explicitly state that GitHub Pages aren't meant for online stores or commercialization like that, but having a Donate button to allow people to support you as a developer is arguably far from commercialization. 

Here is a link to the GitHub Pages documentation: https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages
Here is a link to the GitHub Terms of Service: https://docs.github.com/en/site-policy/github-terms/github-terms-of-service
